cypress/tests/integration/Crossref.spec.js

```diff
@@ -0,0 +1,153 @@
+		cy.waitJQuery();
+		
+		// Go to see if the DOI is assigned
+		cy.get('a:contains("' + issueDescription + '")').click();
+		cy.get('div#editIssueTabs a').contains('Identifiers').click();
+
+		cy.get('fieldset#pubIdDOIFormArea p').contains('The DOI is assigned to this issue.');
+	});
+
+	it('Check Issue DOI Visible', function() {
+		// Chec
+		cy.visit('');
+		cy.get('a:contains("Archives")').click();
+		cy.get('a:contains("' + issueDescription + '")').click();
+		cy.get('div.pub_id').should('have.class', 'doi');
+		cy.get('div.doi span.id a').contains('https://doi.org/10.1234/')
+	});
+
+	it('Check Submission DOI Configuration', function() {
+		cy.login('dbarnes', null, 'publicknowledge');
+
+		cy.visit('/index.php/publicknowledge/workflow/access/' + submissionId);	
+		
+		cy.get('button#publication-button').click();
+		cy.get('button#identifiers-button').click();
+		
+		cy.get('input[name="pub-id::doi"]')
+			.invoke('val')
+			.then(sometext => {
+				if (sometext == null || sometext.trim() == '') {
+					cy.get('input[name="pub-id::doi"]')
+						.parent()
+						.find('button').contains('Assign').click();
+
+					cy.get('div#identifiers')
+						.find('button:contains("Save")').click();	
+
+					cy.get('span').contains('Saved')
+				} else {
+					expect(sometext).to.contain('10.1234');
+				}
+			});
+
+		cy.visit('/index.php/publicknowledge/workflow/access/' + submissionId);	
+		cy.get('button#publication-button').click();
+
+		cy.get("body").then($body => {
+			if ($body.find("button.pkpButton:contains('Publish')").length > 0) {
+				cy.get('button.pkpButton').contains('Publish').click();
+				cy.contains('All publication requirements have been met.');
+					cy.get('.pkpWorkflow__publishModal button').contains('Publish').click();
+			}
+		});
+	});
+
+	it('Check Submission DOI Visible', function() {
+		cy.login('dbarnes', null, 'publicknowledge');
+
+		// Select a submission
+		cy.visit('/index.php/publicknowledge/article/view/' + submissionId);
+
+		cy.get('section.item.doi')
+			.find('span.value').contains('https://doi.org/10.1234/');
+	});
+
+	it('Check Submission Crossref Export', function() {
+		cy.login('dbarnes', null, 'publicknowledge');
+
+		cy.visit('/index.php/publicknowledge/management/importexport/plugin/CrossRefExportPlugin');
+		cy.waitJQuery();
+
+		cy.get('input[name=depositorName]').focus().clear().type('admin');
+		cy.get('input[name=depositorEmail]').focus().clear().type('pkpadmin@mailinator.com');
+
+		cy.get('form#crossrefSettingsForm button:contains("Save")').click();
+		cy.get('div:contains("Your changes have been saved.")');
+
+		cy.reload();
+
+		cy.get('a#ui-id-2').click();
+		cy.get('input#select-1').check();
+		cy.get('input#validation').check();
+
+		cy.get('input#downloadFileEnabled').then(elem => {
+			elem.val('0');
+		});
+
+		cy.get('button#export').click();
+
+		cy.get('input#downloadFileEnabled').then(elem => {
+			elem.val('1');
+		});
+
+		cy.get('div:contains("Registration successful!")');
```

Comment:
Just for future reference you can short this to `cy.contains('Registration Successful');`. :+1: